src/hid-usage-name-overrides.json

```diff
@@ -60,15 +60,16 @@
     "100": { "short": "NUBS" },
     "103": { "short": "=" },
     "133": { "short": "," },
+    "158": { "Name": "Keyboard Non-PC Return"},
     "134": { "short": "=" },
     "176": { "short": "00" },
     "177": { "short": "000" },
     "224": { "short": "Ctrl", "med": "L Ctrl" },
-    "225": { "short": "Shft", "med": "L Shft", "long": "L Shift" },
+    "225": { "short": "⇧Shft", "med": "L Shft", "long": "L Shift" },
```

Comment:
I think it could make sense to have another section for icons, it would be more flexible, like that it would also be possible to switch depending the OS (typically GUI is quite different).